test/javascripts/components/load-more-test.js.es6

```diff
@@ -0,0 +1,28 @@
+import { _setTestParams } from "discourse/lib/eyeline";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("load-more", { integration: true });
+
+componentTest("preview updates with markdown", {
+  template: `
+    {{#load-more selector=".numbers tr" action=(action loadMore)}}
```

Comment:
I think this should work for both cases:

```
action=loadMore
action=(action "loadMore")
```

and in this case `action=loadMore` should be enough

Reply:
Thanks, it makes sense!